eng/common/templates/steps/generate-sbom.yml

```diff
@@ -5,9 +5,9 @@
 
 parameters:
   PackageVersion: 7.0.0
-  BuildDropPath: '$(Build.SourcesDirectory)/artifacts'
+  BuildDropPath: '$(Build.SourcesDirectory)\artifacts'
```

Comment:
Why change `/` to `\`?  [Here](https://github.com/NuGet/NuGet.Client/pull/6441/files#diff-ff6085ab5cf199ecbf9b59ba9a5854fdc078bdc1c7516d16c95f6bff5ecd070aR127) and other places you preferred `/`, even though in the same line `\` was used.

This is a cherry pick, so I'm not expecting a change.  I'm just curious.